clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/UpsertAmazonLoadBalancerAtomicOperation.groovy

```diff
@@ -71,6 +73,9 @@ class UpsertAmazonLoadBalancerAtomicOperation implements AtomicOperation<UpsertA
   @Autowired
   SecurityGroupLookupFactory securityGroupLookupFactory
 
+  @Value("\${aws.autoCreateELBGroup:false}")
```

Comment:
could you add this flag to the AwsConfiguration.DeployDefaults class then autowire that bean in -- that is where the flag for the application security group creation is (the config namespace would then be `aws.defaults.`)